app/services/platforms/facebook.ts

```diff
@@ -179,7 +179,8 @@ export class FacebookService extends StatefulService<IFacebookServiceState> impl
           this.SET_STREAM_URL(info.stream_url);
         }
         return info;
-      });
+      })
+      .catch(() => null);
```

Comment:
What it's catching here?

Reply:
in cases where users have never streamed on the platform i think the data is unpopulated for prefill, i thought `json.data[0]` would just be undefined but it's possible it's throwing an exception instead

Reply:
The `.catch` literally wraps the all lines of code above into `try-catch` construction. We should strive use try/catch only for parts of the code that we suspect can throw exceptions. Otherwise, all this code looks unconfident.